src/main/java/com/arpnetworking/logback/StenoEncoder.java

```diff
@@ -590,6 +606,52 @@ public boolean isSafe() {
     }
 
     /**
+     * Inject bean identifier attributes. This controls whether the the instance identifier and class name are
+     * always injected into each serialized data or context value. The values are always injected for non-loggable
+     * types and this setting allows injection for all types. By default this is false.
+     *
+     * @since 1.9.0
+     *
+     * @param value Whether to inject the bean identifier attributes for all beans irregardless of loggability.
+     */
+    public void setInjectBeanIdentifier(final boolean value) {
+        _injectBeanIdentifier = value;
+    }
+
+    /**
+     * Whether bean identifier attributes are injected. By default this is false.
+     *
+     * @since 1.9.0
+     *
+     * @return True if and only if bean identifier attributes are injected.
+     */
+    public boolean isInjectBeanIdentifier() {
+        return _injectBeanIdentifier;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    protected String encodeAsString(final ILoggingEvent event, final EncodingException ee) {
+        final StringBuilder encoder = new StringBuilder()
+                .append("{\"time\":\"")
+                .append(ISO_DATE_TIME_FORMATTER.format(Instant.ofEpochMilli(event.getTimeStamp())))
+                .append("\",\"name\":\"EncodingException\",\"level\":\"warn\",\"data\":{\"originalMessage\":");
+        safeEncodeValue(encoder, event.getMessage());
+        encoder.append("},\"exception\":{\"type\":\"")
+                .append(ee.getClass().getName())
+                .append("\",\"message\":\"")
+                .append(ee.getMessage().replaceAll("\"", "'"))
+                .append("\",\"backtrace\":[],\"data\":{}},\"context\":");
```

Comment:
No backtrace is being output in the current version either afaik (e.g. it is not included in EncodingException's toString). However, yes, we should ideally add a stack trace here; but it will have to be hand serialized. I'll file it as an issue unless you see this as a blocker/critical. My personal take is that the exception (typically) from Jackson is never as useful as the data that caused the log message to fail (wrt diagnosis and correction). Let me know whow what you think.